src/audits/TableHasAppropriateHeaders.js

```diff
@@ -73,6 +73,32 @@ goog.require('axs.constants.Severity');
         }
         return false;
     }
+	
+    /**
+     * Checks whether a table is a layout table.
+     *
+     * @returns {boolean} Table is a layout table
+     */
+    function tableIsAFilthyLayoutTable(element) {
+        if (element.getAttribute('role') == 'presentation') {
+            return true;
+        }
+
+        if (element.hasAttribute("summary")) {
+            return true;
+        }
+			
+        // empty tables considered layout tables
+        if (!element.childElementCount > 0) {
```

Comment:
`!` is higher precedence than `>` so this amounts to `if ((!element.childElementCount) > 0)` which is probably not what you wanted (though it coincidentally ends up doing the right thing), but you probably want to just do `if (childElementCount == 0)` anyway.